flow/src/test/java/flow/FlowTest.java

```diff
@@ -105,9 +105,9 @@
     listener.flow.set(new Dos());
   }
 
-  @Test public void historyAddAllIsPushy() {
+  @Test public void historypushAllIsPushy() {
```

Comment:
:+1: once travis goes green I'll make just that change and merge.  don't know why it's taking so long.